recipes/panda3d_viewer/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "panda3d_viewer" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 623842303b9f568d80f5a300d35079738f578917916394abaca03ac4fa7c130a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Please take a look at [grayskull](https://github.com/conda-incubator/grayskull). It will give you the updated syntax without all those quotes. BTW, I think you have to remove at least the ones on the `url` field or the bot won't be able to update your recipe.

Reply:
It is done in the last commit